tests/test_dependencies.py

```diff
@@ -98,6 +98,22 @@ def test_contains(deps):
     assert "not.csv" not in deps
 
 
+def test_equals(deps):
+    # empty table vs. empty table
+    _deps = audb.Dependencies()
+    assert _deps == _deps
```

Comment:
Would this assertion not always hold?

Reply:
My guess would be it should depend on your implementation of `__eq__()`.

Reply:
Here is an example:

```python
class NeverEqual(object):
    def __eq__(self, other):
        return False
```
Then you can do:
```python
>>> obj = NeverEqual()
>>> obj == obj
False
```

So the main reason for this test is to make sure, we don't add any unexpected error to the implementation of `__eq__`.

Reply:
True. With an unreasonable implementation of `__eq__`  you can create unintuitive results.